code/ecdsap256sha2/Hacl.Impl.ECDSA.P256SHA256.Signature.Hashless.fst

```diff
@@ -184,46 +184,43 @@ val ecdsa_signature_core: r: felem -> s: felem -> mLen: size_t -> m: lbuffer uin
   (ensures fun h0 flag h1 -> 
     modifies (loc r |+| loc s) h0 h1 /\
     (
-      assert_norm (pow2 32 < pow2 61); 
-      let hashM = H.hash Def.SHA2_256 (as_seq h0 m) in 
-      let z = nat_from_bytes_be hashM % prime_p256_order in 
+      let z = nat_from_bytes_be (as_seq h0 m) % prime_p256_order in 
       let (rxN, ryN, rzN), _ = montgomery_ladder_spec (as_seq h0 k) ((0,0,0), basePoint) in 
       let (xN, _, _) = _norm (rxN, ryN, rzN) in 
       
       let kFelem = nat_from_bytes_be (as_seq h0 k) in 
       as_nat h1 r == xN % prime_p256_order /\
       as_nat h1 s == (z + (as_nat h1 r) * as_nat h0 privKeyAsFelem) * pow kFelem (prime_p256_order - 2) % prime_p256_order /\
       (
-	if as_nat h1 r = 0 || as_nat h1 s = 0 then 
-	  uint_v flag == pow2 64 - 1
-	else 
-	  uint_v flag == 0
+  if as_nat h1 r = 0 || as_nat h1 s = 0 then 
```

Comment:
your editor seems to hesitate between tabs and spaces -- spaces are of course preferred in HACL*, so it's good that you're avoiding tabs, but here indentation seems off